### DISCUSSION TITLE
I absolutely love this mod and what it has to offer. I would love to know if there are plans for a 1.21 version? <3

### DISCUSSION THREAD

**Discussion by alexia8225** [AverageAnime/create-food-1.20.1#41] [Q&A] [answered]
Question, Suggestion

**Comment by AverageAnime** [comment, marked as answer]
The main issue is that Create is not updated to 1.21. The forge version requires Create, so until then it won't be updated. The fabric version doesn't need Create to run, so a 1.21 version is possible. It just wouldn't have any of the Create recipes.
